demisto_sdk/commands/common/content/objects/pack_objects/abstract_pack_objects/json_content_object.py

```diff
@@ -120,7 +123,21 @@ def dump(
             1. Handling case where object changed and need to be serialized.
         """
         created_files: List[Path] = []
-        created_files.extend(super().dump(dest_dir=dest_dir))
+
+        try:
+            created_files.extend(
+                self._unify(
+                    dest_dir=self._create_target_dump_dir(dest_dir=dest_dir),
+                    output=self.normalize_file_name(),
+                )
+            )
+            logger.debug(f"Successfully unified {self.path} {self.type()}")
+        except Exception as e:
+            logger.error(
```

Comment:
maybe debug? it's not necessarily an error, correct me if I'm wrong

Reply:
I agree, can be changed to debug, but @dorschw thinks it should be debug log.

Reply:
If it's not an error, and users won't feel it, let it be `debug`